src/products/mozilla_org/testing.md

```diff
@@ -73,17 +73,15 @@ Updated translations are pushed to the production server regularly throughout th
 
 It is safe to say that it will take less than an hour to see your changes reflected on the production server.
 
-When a project has a firm deadline to meet, it will be communicated through the [dev-l10n-web mailing list](https://lists.mozilla.org/listinfo/dev-l10n-web). Be sure to sign up so you receive important community wide information on web related projects. You can also check out the deadline at page level in Pontoon.
+When a project has a firm deadline to meet, it will be indicated at the top left corner of the project page in Pontoon and at the page level. When necessary, it will be communicated through [the l10n-community room on Matrix]https://chat.mozilla.org/#/room/#l10n-community:mozilla.org). 
 
-## Testing content migrated to Fluent
-
-This section focuses on review and test after a page is migrated to Fluent.
+## Testing content in Fluent
 
 ### Reviewing and updating [brands.ftl](https://github.com/mozilla-l10n/www-l10n/blob/master/en/brands.ftl)
 
-This new file captures all the brand and product names that appear in all the pages on mozilla.org as of now. As a general rule, brands and product names remain unchanged. The exceptions are:
+This file captures all the brand and product names that appear in all the pages on mozilla.org. As a general rule, brands and product names remain unchanged. The exceptions are:
 
-* Account/Accounts in Firefox Account/Accounts
+* Account/Accounts in Mozilla Account/Accounts
```

Comment:
That's misleading, given that "accounts" should be lowercase most of the time. Please sync with Bryan about finding the guidelines to point to.